qiskit/backends/ibmq/credentials/_configrc.py

```diff
@@ -152,3 +152,22 @@ def remove_credentials(credentials, filename=None):
         raise QISKitError('The account "%s" does not exist in the '
                           'configuration file')
     write_qiskit_rc(stored_credentials, filename)
+
+
+def remove_all_credentials(filename=None):
+    """Remove all credentials from qiskitrc.
```

Comment:
why does this need to be in _configrc.py. cant we just have a loop ibmqprovider that calls remove_account for each account.

Reply:
I originally did that and it didn't pass the tests I added. Then I figured instead of trying to figure out why it'd be faster (both in execution and time to implement) to just replace it with an empty file.